dotnet/src/SemanticKernel/KernelConfig.cs

```diff
@@ -87,6 +91,11 @@ public sealed class KernelConfig
     /// <returns>IEnumerable of all embedding service Ids in the kernel configuration.</returns>
     public IEnumerable<string> AllTextEmbeddingGenerationServiceIds => this.TextEmbeddingGenerationServices.Keys;
 
+    /// <summary>
+    /// The REST API operation runner.
+    /// </summary>
+    internal IRestApiOperationRunner? RestApiOperationRunner { get; private set; }
```

Comment:
If we keep adding objects/factories into KernelConfig, we're adding more tech debt about dependency injection, increasing the work we'll have to do to clean up. Why not introduce a generic service collection in the kernel instead?